psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -3107,6 +3223,71 @@ void DFHelper::compute_J(std::vector<SharedMatrix> D, std::vector<SharedMatrix>
         }
     }
 }
+void DFHelper::compute_J_combined(std::vector<SharedMatrix> D, std::vector<SharedMatrix> J, double* Mp, double* T1p, double* T2p, std::vector<std::vector<double>>& D_buffers, size_t bcount, size_t block_size) {
+//    double* coul_metp = metric_prep_core(1.0);
+    double ZERO[16] = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
```

Comment:
Why is this hard-coded to 16 elements and why is it needed?

Reply:
I need to zero out some arrays, and I use C_DCOPY to do so.  A fast copying code might try to copy one cache-length at a time, so I wanted one cache-length worth of memory past an address to be set to zeros in case a C_DCOPY implementation does that.

Reply:
I saw your comment below, so I'll be getting rid of this.